File: CCC/2010/J1.py
n = int(input())

if n < 6:
    if n % 2 == 0:
        print(1 + int(n /2))
    else:
        print(1 + int((n-1) /2))
else:
    if n == 6:
        print(3)
    if n == 7:
        print(2)
    if n == 8:
        print(2)
    if n == 9:
        print(1)
    if n == 10:
        print(10)

File: CCC/2015/J1.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) {
    Scanner input = new Scanner(System.in);
      int month = input.nextInt();
      int day = input.nextInt();

      if (month > 2)
       System.out.println("After");
      else if(month < 2)
       System.out.println("Before");
      else
       if(day == 18)
        System.out.println("Special");
       else if(day < 18)
         System.out.println("Before");
       else
        System.out.println("After");
    }
}
